build/run-tests.sh

```diff
@@ -3,4 +3,4 @@
 set -ex
 
 # shellcheck disable=SC1010
-lein do clean, javac, test :no-gen
+lein do clean, javac, split-test :no-gen
```

Comment:
Within reason, this is a good chance to try running the test suite with `with-profile -dev` which is always cleaner (decomplects dev niceties e.g. a debugger from the bare minimum to run tests)

(leaving this comment just in case it's easy-ish)

Reply:
Seems to work!